department-of-veterans-affairs/caseflow
# Wmedders21/appeals 41493

Resolves [APPEALS-41493](https://jira.devops.va.gov/browse/APPEALS-41493)

# Description
Please explain the changes you made here.
- Standardizes prop names that hold user permission values
- Passes permission values to the dropdown component

**The following changes address code cleanup or test failures in the feature branch.**
- Makes CorrespondenceIntakeTask status `"in_progress"` upon creation
- Abstracts magic strings by referring to constants files
- Removes `task_url` from `EfolderUploadFailedTask` and associated spec expectations.

## Acceptance Criteria

## Testing Plan

1. Log into Caseflow as AMBRISVACO
2. Click on Your Queue, user is redirected to Your cases page.
3. Click on the Switch views dropdown and select Your Correspondence.
4. On Your Correspondence page, click the Switch views dropdown
5. Expected Results: The Superuser has the ability to click the "Switch views" dropdown to display "Correspondence Cases". 

6. Log into Caseflow as AMBRISVACO
7. Click on Your Queue, user is redirected to Your cases page.
8. Click on the Switch views dropdown and select Correspondence Cases.
9. On Correspondence Cases page, click the Switch views dropdown
10. The Superuser has the ability to click the "Switch views" dropdown to display "Your Correspondence" 


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
